pkg/api/transform_filter.go

```diff
@@ -18,20 +18,20 @@
 package api
 
 type TransformFilter struct {
-	Rules []TransformFilterRule `yaml:"rules" doc:"list of filter rules, each includes:"`
+	Rules []TransformFilterRule `yaml:"rules" json:"rules" doc:"list of filter rules, each includes:"`
 }
 
 type TransformFilterOperationEnum struct {
-	RemoveField              string `yaml:"remove_field" doc:"removes the field from the entry"`
-	RemoveEntryIfExists      string `yaml:"remove_entry_if_exists" doc:"removes the entry if the field exists"`
-	RemoveEntryIfDoesntExist string `yaml:"remove_entry_if_doesnt_exist" doc:"removes the entry if the field doesnt exist"`
+	RemoveField              string `yaml:"remove_field" json:"remove_field field from the entry"`
```

Comment:
This line is missing a doc: entry

Reply:
good catch! it's actually malformed, must have been a bad merge conflct resolution